static/styles/night_mode.css

```diff
@@ -49,7 +49,10 @@ body.night-mode {
     .stream_label {
         color: hsl(212, 28%, 18%);
     }
-
+    .user-status-value:hover{
+        color: #467183 !important;
```

Comment:
hey you can do the same thing by adding ( !important ) in the user_status.css file which is also responsible for hover effect in the day-mode.

Reply:
yeah but in day-mode hover effect is working fine. should i add !important in that file?

Reply:
I added !important in the user_status_.css . It works fine in both mode . So I think there is no write the same in another file.

Reply:
If you don't want to make a change then in  place of #467183 write hsl(200, 100%, 40%) it will help you in making all the checks pass.